### DISCUSSION TITLE
Adjust ScrollBar hover style

### DISCUSSION THREAD

**Discussion by halfninja** [AvaloniaUI/Avalonia#17473] [Q&A] [answered]
This may be Fluent theme specific, but the default scroll bars appear as a thin line which widens to the thicker scrollbar if you linger the pointer over it. This is lovely but for clients who find this confusing is it possible to target this through styles to make it always show the larger style whenever it's visible?

![image](https://github.com/user-attachments/assets/9642ddce-9f41-4248-b74e-4564c94561c6)

![image](https://github.com/user-attachments/assets/f205d0cf-d7ce-4c88-b520-538b3010709c)

**Comment by trashhacker** [comment, marked as answer]
try AllowAutoHide property of scrollviewer

**Reply by halfninja** [reply]
That's it! Thanks @trashhacker

**Reply by halfninja** [reply]
Just to confirm it also works on DataGrid by styling the enclosed ScrollBar:

```xaml
<DataGrid.Styles>
  <Style Selector="ScrollBar">
    <Setter Property="AllowAutoHide" Value="False" />    
  </Style>
</DataGrid.Styles>
```

**Reply by trashhacker** [reply]
glad to help u :D
